### DISCUSSION TITLE
No DDC control on M2 Pro/Max

### DISCUSSION THREAD

**Discussion by EwokDiety** [MonitorControl/MonitorControl#1336] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Brightness controls work completely fine throughout the whole range of values, however, the control method is "Software (gamma)". The volume control is unavailable. There is only one slider and that is for the brightness control, there is none for the volume. My monitor's firmware is up to date and DDC is enabled in its menu.

I've tried with multiple cables and I've bypassed my CalDigit TS3 Plus dock and it comes up with the same functionality.

Interestingly, BetterDisplay works just fine for both brightness and volume. I have confirmed that when I use the keyboard buttons to change the display volume that the display's volume magnitude actually changes in the monitor's OSD.

![Screenshot 2023-03-07 at 11 13 26 PM](https://user-images.githubusercontent.com/127280338/223625394-a528c513-3042-4b71-913c-abba0f556b67.jpg)

### Expected behavior

I expected DDC to work with MonitorControl and give me control of both brightness and volume. Volume is what I am most interested in fixing and would love the simplicity of the MonitorControl App.

### Environment Information (please complete the following information)

```markdown
- macOS version: 13.2.1
- Mac model: MacBook Pro 16-inch, 2023 Apple M2 Max
- MonitorControl version: 4.1.0
- Monitor(s): Dell 3818DW
- Monitor Cable(s)/Connection(s): USB-C
- Apple Silicon/M1 (yes or no): YES
```

**Comment by jimothyGator** [comment]
Since you have an M2 Mac, you're likely running into #1299 

The alternative for now is to use [BetterDisplay](https://github.com/waydabber/BetterDisplay)

**Comment by waydabber** [comment]
@jimothyGator is right, M2 is not supported with MonitorControl.
